docs/docs/models-and-databases/reference/query-set.md

```diff
@@ -262,6 +262,13 @@ query_set = Post.all
 query_set.prefetch(:author__favorite_tags)
 ```
 
+In some situations it might be necessary to use a custom query set for the prefetched records. This is possible by providing a `query_set` argument:
```

Comment:
```suggestion
In some situations, it might be necessary to use a custom query set for the prefetched records. This is possible by using a variant of the `#prefetch` method in which a single relation name and the associated query set (`query_set` argument) are provided:
```